clang/lib/Sema/CheckedCAnalysesPrepass.cpp

```diff
@@ -70,16 +82,17 @@ class PrepassHelper : public RecursiveASTVisitor<PrepassHelper> {
       return T->getAsRecordDecl();
     }
 
-    // AddBoundsSiblingField adds FieldWithBounds to the set of fields in
-    // whose declared bounds F occurs.
-    void AddBoundsSiblingField(const FieldDecl *F) {
-      auto It = Info.BoundsSiblingFields.find(F);
-      if (It != Info.BoundsSiblingFields.end())
-        It->second.insert(FieldWithBounds);
+    // AddPtrWithBounds adds PtrWithBounds to the set of variables in whose
+    // bounds Var occurs.
+    template<class M, class P, class V>
+    void AddPtrWithBounds(M &Map, const P *PtrWithBounds, const V *Var) {
```

Comment:
Similarly, could `Var` be renamed since this argument isn't necessarily a variable?

Reply:
Same reply as above. And maybe I can add a note saying "Var may either be a variable or a field".